vanilla_installer/main.py

```diff
@@ -421,28 +417,49 @@ def create_profile(mc_dir: str, version_id: str) -> None:
     launcher_profiles_path.write_text(profiles_json)
 
 
-def get_pack_mc_versions() -> list[str]:
+def get_pack_mc_versions() -> dict:
     """Gets a list of all the versions FO currently supports."""
-    exp = re.compile(r"\d+\.\d+(\.\d+)?")
-    return_value = []
+
+    return_value = dict()
     try:
-        auth = None
-        authdata = get_gh_auth()
-        if authdata is not None:
-            user, key = authdata
-            auth = (user, key)
-        response = requests.get(
-            "https://api.github.com/repos/Fabulously-Optimized/fabulously-optimized/contents/Packwiz",
-            auth=auth,
-        ).json()
-        for response_content in response:
-            if exp.search(response_content["name"]):
-                return_value.append(response_content["name"])
-        return_value.sort()
-        return_value.reverse()
+        try:
+            logger.exception("Couldn't get from the site, falling back to JSDelivr...")
+            response = (
+                requests.get(
+                    "https://raw.githubusercontent.com/Fabulously-Optimized/vanilla-installer/main/docs/meta/versions.json"
```

Comment:
```suggestion
                    "https://raw.githubusercontent.com/Fabulously-Optimized/vanilla-installer/main/vanilla_installer/assets/versions.json"
```

Reply:
:ioa: my ctrl+f didn't find this apparently

Reply:
oh wait, i didn't push that ctrl+f thats why lol